.travis.yml

```diff
@@ -1,25 +1,28 @@
 language: go
 
+dist: bionic
+
 go:
   - 1.10.x
   - 1.11.x
   - 1.12.x
   - 1.13.x
   - tip
 
+git:
+  depth: 1
+
 matrix:
   allow_failures:
     - go: tip
 
-sudo: false
```

Comment:
This used to be the way to specify that you want to run in a container, but [this has been deprecated](https://docs.travis-ci.com/user/reference/overview/#deprecated-virtualization-environments).